base_classes/NXtransformations.nxdl.xml

```diff
@@ -92,6 +92,21 @@
 		for each diffractometer and name the group appropriate to the device.
 		Collecting the motors of a sample table or xyz-stage in an NXtransformation
 		group is equally possible.
+
+
+		Following the section on the general dscription of axis in NXtransformations is a section which 
+		documents the fields commonly used within NeXus for positioning purposes and their meaning. Whenever 
+		there is a need for positioning a beam line component please use the existing names. Use as many fields 
+		as needed in order to position the component. Feel free to add more axis if required. In the description 
+		given below, only those atttributes which are defined through the name are spcified. Add the other attributes 
+		of the full set:
+
+		* vector
+		* offset
+		* transformation_type
+		* depends_on
+      
+		as needed.
```

Comment:
Was it agreed by NIAC to prefer using these names? I sometimes use the name of a motion axis device for example.
Should we also explicitly mention that an `NXlog` can be used in place of a dataset?

Reply:
? These are the standard transformation related attribute types.

At the beginning of the code camp we discussed this issue and decided to move all the older NeXus 
positioning fields into NXtransformations. The use case you describe, an axis name called hugo with a very weird operation, is still possible.

The NXlog is a good point and I see where I can fit this in.

Reply:
Sorry, by "these names" I meant "distance", "height", etc.